core/server/data/schema/schema.js

```diff
@@ -327,6 +327,7 @@ module.exports = {
         email: {type: 'string', maxlength: 191, nullable: false, unique: true, validations: {isEmail: true}},
         name: {type: 'string', maxlength: 191, nullable: true},
         note: {type: 'string', maxlength: 2000, nullable: true},
+        geolocation: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
What kind of structure are we expecting to store here? iirc it was supposed to be a JSON with coordinates, would we store any additional metadata along with that?

Reply:
Expecting a full geolocation object as JSON. Example object:

```js
{
    longitude: '-2.2417',
    city: 'Kidderminster',
    timezone: 'Europe/London',
    accuracy: 200,
    asn: 8468,
    region: 'England',
    organization_name: 'Entanet',
    organization: 'AS8468 Entanet',
    country_code: 'GB',
    ip: '188.39.113.90',
    latitude: '52.375',
    area_code: '0',
    continent_code: 'EU',
    country: 'United Kingdom',
    country_code3: 'GBR'
}
```

Reply:
Think we are safe with 2000 lenght here then. The JSON above is 329 symbols, getting over 2k doesn't seem realistic :thinking: 

Unless all cities/regions/countries start renaming themselves to something like - https://en.wikipedia.org/wiki/Llanfairpwllgwyngyll :sweat_smile:

Reply:
> Unless all cities/regions/countries start renaming themselves to something like 

At which point think we can just stop bothering about geolocation :P